monkey/monkey_island/cc/server_utils/encryption/data_store_encryptor.py

```diff
@@ -38,8 +36,7 @@ def _load_key(self) -> KeyBasedEncryptor:
         return KeyBasedEncryptor(plaintext_key)
 
     def _create_key(self) -> KeyBasedEncryptor:
-        # TODO: Can we just use secrets.token_bytes(DataStoreEncryptor._KEY_LENGTH_BYTES)?
-        plaintext_key = Random.new().read(DataStoreEncryptor._KEY_LENGTH_BYTES)
+        plaintext_key = Fernet.generate_key()
```

Comment:
Why Fernet and not secrets ?

Reply:
Fernet specifies its own key generation specification.